scripts/core/superdesk-api.d.ts

```diff
@@ -284,6 +284,7 @@ declare module 'superdesk-api' {
             archive?: boolean;
             externalsource: boolean;
         };
+        _fetchable?: boolean;
```

Comment:
What's the default? If it's not present in the payload can the item be fetched?

Reply:
There is no default value.
If it's not in the payload, then a fetching condition is determined by editFeaturedImage config.

Reply:
Either naming or logic doesn't make sense for me. If I have a text item, why would `editFeaturedImage` be used to determine if it can be fetched?

What does `editFeaturedImage` mean originally? Is it referring to editing the image itself like crops/rotation? Why would it be related to fetching?

Reply:
Tomas, I think exactly the same :) 
`editFeaturedImage` imo must be renamed/removed and all that logic must be refactored.